Model/Queue/Notification/Consumer.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Adyen\Payment\Model\Queue\Notification;
+
+use Adyen\Payment\Api\Data\NotificationInterface;
+use Adyen\Payment\Helper\Webhook;
+use Adyen\Payment\Logger\AdyenLogger;
+use Exception;
+
+class Consumer
+{
+    public const TOPIC_NAME = "adyen.notification";
+
+    /** @var Webhook $webhookHelper */
+    private $webhookHelper;
+
+    /** @var AdyenLogger $adyenLogger */
+    private $adyenLogger;
+
+    /**
+     * @param Webhook $webhookHelper
+     * @param AdyenLogger $adyenLogger
+     */
+    public function __construct(
+        Webhook $webhookHelper,
+        AdyenLogger $adyenLogger,
```

Comment:
```suggestion
        AdyenLogger $adyenLogger
```

Reply:
Hi @candemiralp, 

Thanks! I have updated the PR and apply the suggested changes.

Reply:
Hi @factorin-j,

Thank you for your valuable contribution. We will merge it after we complete the testing and final reviews.

Best Regards,
Can

Reply:
Hi @candemiralp, 

Great! Thanks